pkg/site/site.go

```diff
@@ -475,6 +480,7 @@ func ReadSiteConfig(siteConfig *corev1.ConfigMap, namespace string, defaultIngre
 	var errs []string
 	var result types.SiteConfig
 	result.Spec.SkupperNamespace = siteConfig.Namespace
+	annotationSeparator, _ := siteConfig.Data[SiteConfigAnnotationSeparator]
 	// TODO: what should the defaults be for name, namespace
```

Comment:
Should a default value be applied here is none is set explicitly?

Reply:
done